Dev-Corner/slides/tool_integration.html

```diff
@@ -968,6 +964,68 @@
 
                     <section data-markdown>
                         <script type="text/template">
+                            <img src="../../shared/images/conda_logo.png" class="no-border"  height=25 />
+                            ### Channels
+
+                            Packages are provided through Channels within Continuum.Anaconda. It's the way to contribute.
+
+                            Galaxy via Conda will search packages in 5 channels: iuc,bioconda,r,defaults,conda-forge
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../../shared/images/conda_logo.png" class="no-border"  height=25 />
+                            ### Quickstart
+                            #### Use Conda outside Planemo
+
+                            <div align="left">
+                            - Install a package
+                            </div>
+
+                            ```bash
+                            conda install bioconductor-deseq2
+                            ```
+
+                            <div align="left">
+                            - Install some packages within an isolate environment
+                            </div>
+
+                            ```bash
+                            $ conda create -n bioconductor-deseq2 bioconductor-deseq2 r-ggplot2
+                            $ conda env list
+                                bioconductor-deseq2   *  ~/miniconda2/envs/bioconductor-deseq2
+                                root                     ~/miniconda2
+                            $ source activate bioconductor-deseq2
+                            (bioconductor-deseq2) $ R
+                            ```
+
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../../shared/images/conda_logo.png" class="no-border"  height=25 />
+                            ### Tips and tricks
+                            #### Because you don't want Conda binaries all the time in your $PATH
```

Comment:
I'm not 100% comfortable with this slide, probably because I'm not 100% sure what are the downsides of having conda always in the $PATH.
Anyway, if we keep it, we have to explain a little why "you don't want Conda binaries all the time in your $PATH"